great_expectations/cli/checkpoint.py

```diff
@@ -3,7 +3,6 @@
 from typing import List
 
 import click
-from ruamel.yaml import YAML
 
```

Comment:
Unused in the file but imported and instantiated. All such instances have been deleted.